### DISCUSSION TITLE
Wrapping vuetify components

### DISCUSSION THREAD

**Discussion by MirasMustimov** [vuetifyjs/vuetify#20763] [Q&A]
Hello, how you guys wrap vuetify components while preserving all props/slots/events with TS support?

For example I want a ConfirmButton component which wraps VBtn
and i want the component to be able to accept all props VBtn accepts (with ts support) while having the defaults that Vbtn has.

I know I can just define part of Vbtn props in my ConfirmButton, but i dont want to limit myself to certain subset of props.
Is there a way to just make my component have all the same props?

Is the wrapping components idea valid in vue js in general? I dont see this idea discussed often

**Comment by ChristopherJohnson25** [comment]
What is your main reason for wrapping the component? Why not just leverage Vuetify alias?
